flowers/trainer/task.py

```diff
@@ -300,7 +299,7 @@ def save_summaries(self, session):
     self.sv.summary_writer.flush()
 
 
-def main(_):
+def run_main():
```

Comment:
perhaps consider putting argsparse under `if __name__ == '__main__':` and pass `args` to `def main(args)`.

(please ignore if such changes are out of scope)